### ISSUE TITLE
loss of test set in training process

### ISSUE THREAD

**Issue by Phil-Lin** [jswati31/gazeclr#2]
hi, When I was training the model normally, I found that the loss of the training set dropped normally, but the loss of the test set did not drop too much. Have you ever encountered such a situation in your training? I see your initial code is to turn off the test set of tests.
![1680832534844](https://user-images.githubusercontent.com/52906245/230524845-481072f3-918a-4f4d-a967-735529b42d8d.jpg)

![1680832482946](https://user-images.githubusercontent.com/52906245/230524735-9e301509-77b0-4a8c-873e-02917771e309.jpg)

**Comment by jswati31** [comment]
Yes, the test loss doesn't change much. It was computed to verify if the training behavior is normal and is not nan.
The training is performed for fix number of iterations without using any validation loss.

**Comment by Phil-Lin** [comment]
Thank you for your reply，I have another question. In the line 108 to 116 of  EveDaset.py, you have changed the order of the samples, but you haven't changed the order of the parameters 'face_g_tobii'、‘camera_transformation’ and ‘face_R’, which maybe lead to errors in the final calculation of losses.

**Comment by jswati31** [comment]
Thanks for pointing this out. I have updated the code.
